test/ao/sparsity/test_parametrization.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from torch import nn
+from torch.ao.sparsity import utils
+from torch.nn.utils import parametrize
+
+import torch
+from torch.testing._internal.common_utils import TestCase
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)
```

Comment:
Is this the way logging is done in pytorch?

Reply:
yes -- for unittests